solr/core/src/test/org/apache/solr/cli/ZkSubcommandsTest.java

```diff
@@ -376,26 +377,26 @@ public void testUpConfigLinkConfigClearZk() throws Exception {
     assertEquals(confsetname, collectionProps.getStr("configName"));
 
     // test down config
-    Path configSetDir =
-        tmpDir.resolve(
-            "solrtest-confdropspot-" + this.getClass().getName() + "-" + System.nanoTime());
-    assertFalse(Files.exists(configSetDir));
+    Path destDir =
```

Comment:
A clearer name, as this test deals with many configSetDirs, as input and output.